pallets/crowdloan-reward/src/mock.rs

```diff
@@ -0,0 +1,263 @@
+// Copyright 2021 Centrifuge GmbH (centrifuge.io).
+// This file is part of Centrifuge chain project.
+
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+//! Crowdloan reward pallet testing environment and utilities
+//!
+//! The main components implemented in this module is a mock runtime
+//! and some helper functions.
+
+#![cfg(test)]
+
+// ----------------------------------------------------------------------------
+// Imports and dependencies
+// ----------------------------------------------------------------------------
+
+use frame_support::{parameter_types, traits::SortedMembers, weights::Weight, PalletId};
+use frame_system::EnsureSignedBy;
+use sp_core::H256;
+use sp_runtime::{
+    testing::Header,
+    traits::{BlakeTwo256, IdentityLookup},
+    Perbill,
+};
+
+use crate as pallet_crowdloan_reward;
+use crate::traits::WeightInfo;
+
+// ----------------------------------------------------------------------------
+// Mock runtime
+// ----------------------------------------------------------------------------
+
+// Extrinsics weight information used for testing
+pub struct MockWeightInfo;
```

Comment:
cant we just get away with `()` implementing weightInfo ?

Reply:
Done!